ivadomed/keywords.py

```diff
@@ -274,3 +276,58 @@ class SegmentationPairKW:
     INPUT_METADATA = "input_metadata"
     GT = "gt"
     INPUT = "input"
+
+@dataclass
+class CommandKW:
+    TRAIN = "train"
+    SEGMENT = "segment"
+    TEST = "test"
+
+@dataclass(frozen=True)
+class DataloaderKW:
+    # AllDatasetGroups Level
+    DATASET_GROUPS: str = "dataset_groups"
+
+    # DatasetGroup Level
+    DATASET_GROUP_LABEL: str = "dataset_group_label"
+    TRAINING: str = "training"
+    VALIDATION: str = "validation"
+    TEST: str = "test"
+    # Reserved DatasetGroup to be implemented
+    TRAINING_VALIDATION: str = "training_validation"
+    TRAINING_TEST: str = "training_test"
+
+    # Dataset Level
+    TYPE: str = "type"
+    DATASET_LABEL: str = "dataset_label"
+    IMAGE_GROUND_TRUTH: str = "image_ground_truth"
+    EXPECTED_INPUT: str = "expected_input"
+    EXPECTED_GT: str = "expected_gt"
+    # Reserved keyword for future implementation
+    MISSING_FILES_HANDLE: str = "missing_files_handle"
+    EXCESSIVE_FILES_HANDLE: str = "excessive_files_handle"
+
+    SUBSET_LABEL: str = "subset_label"
+    PATH_DATA: str = "path_data"
+
+
+@dataclass(frozen=True)
+class DatasetTypeKW:
+    # Used to specify the datatype in the display_selected_transfoms
```

Comment:
Just converting the comment into a doc string would be a little more pythonic...

Reply:
Good point. Will make these changes.